source/guides/network/openssh-server.rst

```diff
@@ -104,32 +104,20 @@ Enable SFTP
 ***********
 
 |CL| *disables* the :abbr:`SFTP (SSH File Transfer Protocol)` subsystem by
-default due to security considerations. To enable the SFTP subsystem, you must
-configure the :abbr:`SSHD (SSH Daemon)` service file.
+default due to security considerations. To enable the SFTP subsystem, you can
+configure the :file:`/etc/ssh/sshd_config` file.
 
-#. Create a systemd drop-in directory for the SSHD service:
-
-   .. code-block:: bash
-
-      sudo mkdir -p /etc/systemd/system/sshd@.service.d
-
-#. Create the following file:
-   :file:`/etc/systemd/system/sshd@.service.d/sftp.conf`
+#. Create the following file, if it does not already exist:
+   :file:`/etc/ssh/sshd_config`
 
-#. Add the OPTIONS environment variable to the :file:`sftp.conf` file.
+#. Add the the SFTP subsystem in :file:`/etc/ssh/sshd_config`:
 
    .. code-block:: console
 
-      [Service]
-      Environment="OPTIONS=-o Subsystem=\"sftp /usr/libexec/sftp-server\""
+      subsystem sftp /usr/libexec/sftp-server
 
-#. Reload systemd configuration:
 
-   .. code-block:: bash
-
-      sudo systemctl daemon-reload
-
-Congratulations! The SFTP subsystem is enabled.
+Congratulations! The SFTP subsystem is enabled. You do not need to restart the sshd service.
```

Comment:
I would add this note to the root login section as well.

Reply:
Good call -- adds balance.  Will add with next commit.